src/porepy/numerics/linalg/matrix_operations.py

```diff
@@ -1062,3 +1062,101 @@ def sparse_array_to_row_col_data(
         return (mat_copy.row[nz_mask], mat_copy.col[nz_mask], mat_copy.data[nz_mask])
     else:
         return (mat_copy.row, mat_copy.col, mat_copy.data)
+
+
+def invert_permutation(perm):
+    """Invert permutation array.
+
+    Parameters:
+        perm : permutation array
+
+    Returns:
+        inv_perm: Permuted sparse array
+
+    """
+
+    x = np.empty_like(perm)
+    x[perm] = np.arange(len(perm), dtype=perm.dtype)
+    return x
+
+
+def sparse_permute(
+    a: sps.spmatrix, row_perm: np.ndarray, col_perm: np.ndarray, inplace_q: bool = False
+) -> sps.spmatrix:
+    """Permute a sparse array.
+
```

Comment:
```suggestion

For a matrix A, this function is equivalent to calling A[row_perm][:, col_perm], however, the present function
should be faster.
```
Is this explanation correct?